pkgs/data/icons/polar-xcursor-theme/default.nix

```diff
@@ -0,0 +1,45 @@
+{ stdenv, lib, fetchurl, variants ? [ "Blue" "Green" ] }:
+
+with lib;
+
+stdenv.mkDerivation rec {
+  name = "${package-name}-${version}";
+  package-name = "polar-xcursor-theme";
+  version = "1.4";
+
+  src = fetchurl {
+    url = "https://dl.opendesktop.org/api/files/download/id/1460735356/27913-PolarCursorThemes.tar.bz2";
```

Comment:
The hash of this theme changed to `167m2q6fb1warzwr836jbhjdiddhzhnzskhd4kmhgxc9ii97rmq3`.
Is there a more stable url?